docs/cryoet_data_portal_docsite_quick_start.md

```diff
@@ -33,13 +34,56 @@ Install the latest `cryoet_data_portal` package via pip:
 pip install -U cryoet-data-portal
 ```
 
-## Python quick start
+## API Functions Overview
 
-Below are 3 examples of common operations you can do with the client. Check out the [examples page](https://chanzuckerberg.github.io/cryoet-data-portal/cryoet_data_portal_docsite_examples.html) for more code snippets.
+The Portal API has functions for searching and downloading data. **Every class** has a `find` and `get_by_id` method for selecting data, and most classes have `download...` methods for downloading the data. Below is a table of the API classes download functions.
 
-### Browse data in the portal
+| **Class**               | **Download Functions**                                                                                 |
```

Comment:
Can we also call them `Download Methods` here like in the text above?

Reply:
Agreed. I updated here and everywhere else I used the word function.